NumpyXBench/operators/common_ops.py

```diff
@@ -6,12 +6,21 @@
     import torch
 except ImportError:
     pass
+
 from jinja2 import Template
 
 __all__ = []
 
 common_op_list = ['add', 'subtract', 'multiply', 'divide']
 
+module_switcher = {
```

Comment:
add `common.py` in utils and move this to there

Reply:
also use it in `benchmark`

Reply:
Done